### ISSUE TITLE
Timezone in prometheus dashboard

### ISSUE THREAD

**Issue by ravi-kp** [jupp0r/prometheus-cpp#553]
Is it possible to get local time support instead of UTC in the Prometheus dashboard?

Let's I want to make data available at the metric endpoint along with its timestamp(the time at which we got this data). But the system time is in PDT and **I also want the Prometheus dashboard x-axis with this PDT time, not in(UTC or equivalent local time).**

In go there is an API  NewMetricWithTimestamp(time, metric)
but here time is in UTC so I need to convert my PDT time to UTC and then this API returns the data value with the timestamp in milliseconds. So on the Prometheus dashboard, we are able to see the equivalent UTC time.

**Comment by ravi-kp** [comment]
Can anyone have a look into the issue, please?

**Comment by gjasny** [comment]
Hello,

your problem seems to be related to the [prometheus](https://github.com/prometheus/prometheus) project, not `prometheus-cpp`.

Thanks,
Gregor

**Comment by ravi-kp** [comment]
yes got it, Thanks. but is there any API in C++ to expose metric value along with some timestamp at metrics endpoint?
